src/GraphQL.Tests/Utilities/Visitors/UppercaseDirectiveVisitor.cs

```diff
@@ -51,13 +46,11 @@ public override void VisitObjectFieldDefinition(FieldType field, IObjectGraphTyp
             if (applied != null)
             {
                 var inner = field.Resolver ?? NameFieldResolver.Instance;
-                field.Resolver = new AsyncFieldResolver<object>(async context =>
+                field.Resolver = new FuncFieldResolver<object>(async context =>
```

Comment:
Compare UppercaseDirectiveVisitor with AsyncUppercaseDirectiveVisitor. It's time to remove the latter :)

Reply:
Done